Sources/xcodeproj/Objects/Project/PBXObjects.swift

```diff
@@ -339,6 +339,20 @@ extension PBXObjects {
         phases.merge(frameworksBuildPhases as [PBXObjectReference: PBXBuildPhase], uniquingKeysWith: { first, _ in return first })
         return phases
     }
+    
+    var buildPhaseFile: [PBXObjectReference: PBXBuildPhaseFile] {
```

Comment:
I'd add a comment here with some context on why this dictionary is necessary. Did you check if the creation of this dictionary has a significant impact on memory consumption?

Reply:
Yes, I checked, the effect is insignificant (I did not notice it at all), since these are calculated properties and all this data is already in memory.